game/hud/src/services/session/layout.ts

```diff
@@ -109,18 +121,80 @@ const initialState = {
   widgets: defaultWidgets(),
 }
 
-let getInitialState = () => {
-  var storedState = JSON.parse(localStorage.getItem(localStorageKey)) as LayoutState;
+interface Size {
+  width: number;
+  height: number;
+}
+
+const px2pcnt = (pos: Position, screen: Size) : Position => {
```

Comment:
8.1 is talking about the use of arrow functions not the definition of functions via arrows. It does not allow it.

However JB has said that the existing conventions of of the codebase are king so it is allowed